processor/src/test/java/org/mapstruct/ap/test/bugs/_913/Issue913SetterMapperForCollectionsTest.java

```diff
@@ -173,6 +175,7 @@ public void shouldReturnDefaultForNvmsReturnDefaultForUpdate() {
         Dto dto = new Dto();
         Domain domain = new Domain();
         Set<Long> longIn = new HashSet<Long>();
+        longIn.add( 10L );
```

Comment:
is this a testcase improvement or did I miss something? (Adding values to the set).